README.md

```diff
@@ -15,6 +15,56 @@ List of available parameters:
 
 - `GATSBY_API_URL`: override api url (defaults to location origin)
 
+### MongoDB Setup
+
+Mongo needs a couple of extra steps in order to start a secure cluster.
+
+* Open port 27017 to all nodes that will take part in the cluster.
```

Comment:
```suggestion
* Open port 27017 on all nodes that will take part in the cluster.
```

Reply:
I actually meant `to` but I ended up doing `on`, haha. What I mean is that my original idea was to allow connections to 27017 only from the other nodes that are part of this replica set. But there was something wrong with that and the cluster didn't work, so I switched to "everybody can connect to 27017" for the moment. This is not great and we should revisit it later.